.github/workflows/e2ePerformanceTests.yml

```
@@ -205,6 +205,23 @@ jobs:
           cat ./mainResults/Host_Machine_Files/\$WORKING_DIRECTORY/debug.log || true
           cat "./mainResults/Host_Machine_Files/\$WORKING_DIRECTORY/Test spec output.txt" || true
 
+      - name: Announce failed workflow in Slack
+        if: ${{ failure() }}
```

In the step above we check the failure like this, does it not matter?
```suggestion
        if: failure()
```

---

oh, good point the double curly brackets are actually not needed - removed them!